### PR TITLE
adds a mongodb query for alertable data

### PR DESCRIPTION
**Pull request by ewollesen**
Where alertable data means those data used when evaluating whether a defined alert configuration is triggered, and a notification needs to be sent.

BACK-2554

### CODE REVIEW

File: data/types/alertable/alertable.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package alertable
+
+// Package alertable extracts types used when querying data used to evaluate
+// alerts configurations.
+//
+// I tried to put these in the data/types package (as AlertableParams and
```
Comment (toddkazakov) [inline data/types/alertable/alertable.go:6]: I don't think this package should be under `data/types`. All other types in the package are device data structs.

Comment (ewollesen) [reply data/types/alertable/alertable.go:6]: > I don't think this package should be under `data/types`. All other types in the package are device data structs.

Where would you like to see it live?

Comment (ewollesen) [reply data/types/alertable/alertable.go:6]: I'll move it to data/store... is that better? Just let me know.

Comment (toddkazakov) [reply data/types/alertable/alertable.go]: `data/store` is probably ok

**Review by toddkazakov — APPROVED** [review]

**Comment by ewollesen** [comment]
@tjotala how does this fit with QA? This is 100% new functionality, nothing to test for QA.

Todd (in slack) said:
> IMO, unit tests are fine. That’s new functionality, nothing is using it, no unrelated changes. As long as it builds, it should be fine, but please confirm with 
@tjotala 

I'm glad to do whatever folks think is best and fits with our processes.

**Review by tjotala — APPROVED** [review]
LGTM to merge as incremental work that does not affect current functionality. Will be tested along with the CPA work.

**Pull request merged**